### PR TITLE
Unit Test for MapUtil

### PR DESCRIPTION
**Pull request by mrudultora**
## Purpose / Description
Added a unit test for MapUtil.

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [x] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Review by Arthur-Milchior — APPROVED** [review]

File: AnkiDroid/src/test/java/com/ichi2/utils/MapUtilTest.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+    @Test
+    public void getKeyByValueIsEqualTest() {
+        assertThat(getKeyByValue(map, "AnkiDroid"), is(20));
+    }
+
+
+    @Test
+    public void getKeyByValueIsNotEqualTest() {
+        assertThat(getKeyByValue(map, "AnkiDesktop"), not(5));
```
Comment (david-allison-1) [inline AnkiDroid/src/test/java/com/ichi2/utils/MapUtilTest.java:49]: It'd be useful to define what happens in the edge cases:

* If map is null
* If the value is null
* If the value exists multiple times
* If the value does not exist

Comment (mrudultora) [reply AnkiDroid/src/test/java/com/ichi2/utils/MapUtilTest.java:49]: Should I make a new PR for that? Because this one is now merged.

Comment (david-allison-1) [reply AnkiDroid/src/test/java/com/ichi2/utils/MapUtilTest.java:49]: Oh, sure!
